client/lib/like-store/actions.js

```diff
@@ -82,6 +82,14 @@ export function unlikePost( siteId, postId ) {
 		siteId: siteId,
 		postId: postId,
 	} );
+
+	wpcom
+		.site( siteId )
+		.post( postId )
+		.like()
+		.add( getQuery(), function( error, data ) {
```

Comment:
shouldn't that be `del`? :D

Reply:
I really do believe an elf somewhere is messing with me on this one